Classes/Route.php

```diff
@@ -0,0 +1,133 @@
+<?php
+session_start();
+
+class Route
+{
+    public static $configurations = [];
+
+    public static $routes = [];
+
+    /**
```

Comment:
Here must be a description of the method. But for a private function, you can ignore the commenting method. We comment only public methods.
The same for others